doc/level1.rst

```diff
@@ -275,6 +275,23 @@ above rules, such headers will be judged invalid, and discarded by
 level 1 MUAs.  Such an update to the specification will also have
 to describe how an updated MUA will deal with multiple valid headers.
 
+A MUA MUST NOT send Autocrypt headers above 10 KiB in size
+(including the Autocrypt: prefix and folding whitespace).
+Conversely, MUAs SHOULD not process Autocrypt headers
+larger than 10 KiB in size.
+
+.. note::
+
+  - An Autocrypt header following this specification
+    will be up to 3 KiB large.
```

Comment:
we say "in size" everywhere else.  maybe s/large/in size/ here too?

Reply:
sorry... had forgotten to push. Should now actually be resolved.